src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -339,19 +345,37 @@ private void notifyUser(MediaRequestEntity mediaRequestEntity,
             if (notificationTemplateName.equals(NotificationApi.NotificationTemplate.ERROR_PROCESSING_AUDIO.toString())) {
 
                 List<DefendantEntity> defendantList = mediaRequestEntity.getHearing().getCourtCase().getDefendantList();
+
                 List<String> defendantNames = new ArrayList<>();
-                for (DefendantEntity defendant : defendantList) {
-                    defendantNames.add(defendant.getName());
+                String defendants = NO_DEFENDANTS;
+                if (!defendantList.isEmpty()) {
+                    for (DefendantEntity defendant : defendantList) {
+                        defendantNames.add(defendant.getName());
+
+                    }
+
+                    defendants = defendantNames.stream().collect(Collectors.joining(","));
```

Comment:
might be best to have a space next to the comma? i.e. ", "

Reply:
Might be worth adding a very simple unit test to cover the scenario with multiple defendants and check the output of this line is correctly formatted?

Reply:
also is it cleaner to use this?:-
```java
String.join(", ", defendantNames)
```